### PR TITLE
Web tests to Github Actions

### PR DESCRIPTION
**Pull request by himadrisingh**

### CODE REVIEW

File: .github/workflows/web-test.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+name: Build and Test web code
+on:
+  pull_request:
+    paths:
+      - ".github/workflows/web-test.yml"
+      - "web-admin/**"
+      - "web-cloud/**"
```
Comment (begelundmuller) [inline .github/workflows/web-test.yml:7]: This should be `web-common`

File: .github/workflows/web-test.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          node-version: 18
+
+      - name: NPM Install
+        run: npm install
+
+      - name: Disabling telmetry
+        run: cat <<< "$(jq -c -M '.sendTelemetryData |= false' ~/.rill/local.json)" > ~/.rill/local.json
+
+      - name: Prettier checks
+        run: npx prettier --check "**/*"
```
Comment (begelundmuller) [inline .github/workflows/web-test.yml:28]: I've run into prettier checking `yaml` files in the entire repo formerly. If someone edits examples, they won't get an error, and can merge into main. Then next time someone edits `web` paths, they'll get the broken CI instead. Options:

1. Reconfigure prettier to only check files in web paths
2. Setup prettier as a separate job that runs on all paths

**Review by begelundmuller — CHANGES_REQUESTED** [review]

File: .github/workflows/web-test.yml (hunk 1 above)
Comment (himadrisingh) [reply .github/workflows/web-test.yml:7]: oh yeah, for some reason in my local there is web-cloud too

File: .github/workflows/prettier.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+name: Build and Test web code
```
Comment (begelundmuller) [inline .github/workflows/prettier.yml:1]: Update title here to prevent confusion

**Review by begelundmuller — APPROVED** [review]
one small nit, otherwise looks good!

File: .github/workflows/prettier.yml (hunk 3 above)
Comment (himadrisingh) [inline .github/workflows/prettier.yml:1]:
```suggestion
name: Prettier check
```

**Pull request merged**